### PR TITLE
Lps x 8 9 1

### PR DESCRIPTION
**Pull request by W2jx**
Hi @dantewang , I and @JanisZhang finished the modification about "*PanelApp.java".

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:sf

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:acceptance

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/application/list/AccountEntriesAdminPanelApp.java [hunk 1]
```diff
@@ -36,18 +36,20 @@
...
 
-	@Override
 	@Reference(
 		target = "(javax.portlet.name=" + AccountPortletKeys.ACCOUNT_ENTRIES_ADMIN + ")",
 		unbind = "-"
 	)
-	public void setPortlet(Portlet portlet) {
-		super.setPortlet(portlet);
-	}
+	private Portlet _portlet;
```
Comment (dantewang) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/application/list/AccountEntriesAdminPanelApp.java:53]: @W2jx Field reference does not need this `unbind = "-"`.

**Review by dantewang — CHANGES_REQUESTED** [review]
@W2jx Other parts looks good, just one thing, please see my comment.

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/application/list/AccountEntriesAdminPanelApp.java (hunk 1 above)
Comment (W2jx) [reply modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/application/list/AccountEntriesAdminPanelApp.java:53]: I will fix it.

**Comment by W2jx** [comment]
Hi @dantewang , there is an error when I execute ant all， For example PortletPanelAppAdapter has no portlet variable, it extends BasePanelApp and uses the method getPortlet in BasePanelApp，but we remove the method getPortlet in BasePanelApp

**Comment by dantewang** [comment]
@W2jx OK I see. The setPortlet method is not only used by sub-classes. They are also used by some trackers.

I think this is somehow a backwards compatible thing for the adapter usages. If you register a PanelApp using OSGi component, you should not need the PanelAppRegistry to inject the portlet for you.

Please push `setPortlet` method to the adapter, and only `setPortlet` if the `PanelApp` instance is an adapter instance.

**Pull request closed without merging**